docs/faq/migration/token-migration.md

```diff
@@ -0,0 +1,74 @@
+
+### Will it be possible to migrate LYX back to LYXe?
+
+The official [LUKSO Migration Bridge](https://migrate.lukso.network/) only acts as a 1-Way-Bridge to migrate LYXe to LYX. Other bridges might exist in the future but are not officially supported by the team.
+
+### What is the minimal amount of LYX that can be bridged?
+
+You need at least 1 LYXe within your connected wallet to start the migration. You can also send comma values.
+
+### How much will it cost to migrate the LYXe?
+
+Since LYXe is managed on Ethereum, you will need some ETH to cover the Gas costs of the transaction when using the migration dApp. The transfer will cost less than 200k Gas, corresponding to [an average](https://etherscan.io/gastracker) of 0,004 ETH. Remember that these prices can fluctuate if the network is at capacity.
+
+### Which wallets can be used to migrate LYXe?
+
+Only regular EVM-compatible browser-based wallets (Externally Owned Accounts) are supported for the migration. Universal Profiles are disabled, so your funds are kept anonymous and not directly linked to any public persona.
+
+### Where will the LYX be migrated to?
+
+The LYX will be sent to the same blockchain address used during the migration. Within supported exchanges, the accounts will automatically be updated from LYXe to LYX if they have not been manually transferred to external wallets.
+
+### How do I migrate if I have LYXe on a hardware wallet?
+
+If you have LYXe on a hardware wallet like Ledger or Trezor, you can send them to a supported crypto exchange or migrate independently by connecting your hardware wallet to a browser-based wallet. Make sure your hardware wallet supports external connections to browser-based wallets.
+
+### What if I don't have a browser-based wallet?
+
+If you migrate yourself but don't have a browser-based wallet, e.g., holding your LYXe on a Ledger or Trezor, we recommend installing an EVM-compatible browser-based wallet. After generating a new blockchain address on the installed wallet, you can send the LYXe to your new wallet and begin migrating from the browser. One of the most commonly used browser-based wallets is [MetaMask](https://metamask.io/download/), which can be installed and added to your browser within a few steps. MetaMask also supports wallet imports for both Ledger and Trezor hardware wallets.
+
+### How can I see my LYX balance?
+
+After completing the process, you can add the network from the migration dApp. If you want to see your balances without going through the migration process, you can add the RPC endpoint of one of the supported service providers like [GatewayFM](https://gateway.fm/). Have a look at the [Network Properties](https://docs.lukso.tech/networks/mainnet/parameters) to see the connection details.
```

Comment:
ppl can also check directly on the blockchain explorer:

https://explorer.execution.testnet.lukso.network/

might be much easier than adding custom network etc.

--

you can use this link: https://gateway.fm/lukso/ instead of https://gateway.fm/